test/modules.coffee

```diff
@@ -183,6 +183,28 @@ test "multiline complex import", ->
     } from 'lib';"""
   eq toJS(input), output
 
+test "import with optional commas", ->
+  input = "import { foo, bar, } from 'lib'"
+  output = """
+    import {
+      foo,
+      bar
+    } from 'lib';"""
+  eq toJS(input), output
+
+test "multiline import with optional commas", ->
+  input = """
+    import {
+      foo,
+      bar,
```

Comment:
Nice! Extra trailing comma is now accounted for. We also need to test _removed_ commas:

```coffee
import {
  foo
  bar
} from 'lib'
```